elixir/apps/web/assets/js/hooks.js

```diff
@@ -29,7 +29,6 @@ const statusUpdater = function () {
 }
 Hooks.StatusPage = {
   mounted: statusUpdater,
-  updated: statusUpdater,
```

Comment:
I've tested and this will not work, if you start typing a form which live validation the status page widget will just disappear. I have a fix that uses `phx-update=ignore` in my branch, so don't worry about this one.

Reply:
Gotcha. Thanks for the fix.